service/src/test/java/bio/terra/workspace/app/configuration/external/controller/WorkspaceApiControllerTest.java

```diff
@@ -67,6 +68,7 @@
  * currently in WorkspaceServiceTest, it's intended as a proof-of-concept for future mockMvc-based
  * tests.
  */
+@Disabled("PF-1962 This test needs to be rewritten as a connected test since it makes a real GCP cloud context")
```

Comment:
I don't think this test makes real cloud contexts either, the tests create workspaces but never calls the GCP cloud context endpoints